docs/reference/migration/apis/deprecation.asciidoc

```diff
@@ -49,20 +49,20 @@ Example response:
 {
   "cluster_settings" : [
     {
-      "level" : "info",
-      "message" : "Network settings changes",
-      "url" : "{ref-60}/breaking_60_indices_changes.html#_index_templates_use_literal_index_patterns_literal_instead_of_literal_template_literal",
-      "details" : "templates using `template` field: watches,.monitoring-alerts,.watch-history-6,.ml-notifications,security-index-template,triggered_watches,.monitoring-es,.ml-meta,.ml-state,.monitoring-logstash,.ml-anomalies-,.monitoring-kibana"
+      "level" : "critical",
+      "message" : "Cluster name cannot contain ':'",
+      "url" : "{ref-60}/breaking-changes-7.0.html#_literal_literal_is_no_longer_allowed_in_cluster_name",
+      "details" : "This cluster is named [mycompany:logging], which contains the illegal character ':'."
     }
   ],
   "node_settings" : [ ],
   "index_settings" : {
-    ".monitoring-es-6-2017.07.21" : [
+    "logs:apache" : [
       {
-        "level" : "info",
-        "message" : "Coercion of boolean fields",
-        "url" : "{ref-60}/breaking_60_mappings_changes.html#_coercion_of_boolean_fields",
-        "details" : "[[type: doc, field: spins], [type: doc, field: mlockall], [type: doc, field: node_master], [type: doc, field: primary]]"
+        "level" : "warning",
+        "message" : "Index name cannot contain ':'",
+        "url" : "{ref-60}/breaking-changes-7.0.html#_literal_literal_is_no_longer_allowed_in_index_name",
```

Comment:
I believe `{ref-60}` should be changed to `{ref}` for 7.0 changes since I cannot find this page in 6.x